src/main/java/com/meilisearch/sdk/TasksHandler.java

```diff
@@ -48,9 +48,7 @@ public Task getTask(String indexUid, int taskUid) throws MeilisearchException {
     public Result<Task> getTasks(String indexUid) throws MeilisearchException {
         String urlPath = "/indexes/" + indexUid + "/tasks";
 
-        Result<Task> result =
-                httpClient.jsonHandler.decode(
-                        this.httpClient.get(urlPath), Result.class, Task.class);
+        Result<Task> result = httpClient.get(urlPath, Result.class, Task.class);
```

Comment:
Same comment as I sent below :)

Reply:
Same I hesitate because it is a complex type and this is the line that generates the `Type safety` warning